packages/fast-components-styles-msft/src/select/index.ts

```diff
@@ -24,24 +24,21 @@ import { inputFieldStyles } from "../patterns/input-field";
 
 const styles: ComponentStyles<SelectClassNameContract, DesignSystem> = {
     select: {
-        minWidth: "276px",
+        display: "inline-block",
```

Comment:
What was this before - block? Are we worried about breaking layouts changing this?

Reply:
@nicholasrice I think that's a fair point; I can remove this change if we want to hold off on this update until vNext